lib/evss/pciu_address/address_domestic.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module EVSS
+  module PCIUAddress
+    class AddressDomestic < Address
```

Comment:
Rename to `DomesticAddress`?  Feel free to ignore, it's just how I would say the term in plain language.